src/hooks/useSiteTree.tsx

```diff
@@ -0,0 +1,28 @@
+import { useAuth } from '../components/Auth/Auth';
+
```

Comment:
I thkn the site treet item type can be specified here.

Reply:
✅ => 🏷️ ADD type